packages/web/src/scss/components/UNSTABLE_Header/README.md

```diff
@@ -117,5 +117,62 @@ If you need the whole Header fluid you can do it by adding the `Container--fluid
 </header>
 ```
 
+## With Navigation
+
+You can use the [`Navigation`][web-navigation] component inside the `UNSTABLE_Header` component.
+
+The `NavigationLink` components will inherit the `UNSTABLE_Header` height and set the navigation
+link height to the same value.
+
+Use the composition mentioned above to create the layout you need.
+
+```html
+<header class="UNSTABLE_Header">
+  <div class="Container">
+    <div
+      class="Flex Flex--row Flex--noWrap Flex--alignmentXLeft Flex--alignmentYCenter"
+      style="--flex-spacing-x: var(--spirit-space-1000);"
+    >
+      <a href="#" aria-label="JobBoard homepage" class="UNSTABLE_HeaderLogo">
+        <div class="ProductLogo">{{> web/assets/jobBoardLogo }}</div>
+      </a>
+
+      <nav class="Navigation">
+        <ul>
+          <li>
+            <a href="#" class="NavigationLink">Link</a>
+          </li>
+          <li>
+            <a href="#" class="NavigationLink NavigationLink--selected" aria-current="page">Selected</a>
+          </li>
+          <li>
+            <a href="#" class="NavigationLink NavigationLink--disabled">Disabled</a>
+          </li>
+        </ul>
+      </nav>
+
+      <nav class="Navigation ml-auto">
```

Comment:
In case there are two `<nav>` elements (and there can be more on the page), we should add `aria-labelledby` to them, what do you think?

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav#usage_notes

Reply:
We have `aria-label` in the current `Header`:

```html
<nav class="HeaderDesktopActions" aria-label="Main navigation">
  <!-- Desktop-only actions -->
</nav>
```

Reply:
Added aria-label everywhere.